crm-project/src/main/java/CRM/entity/requests/UpdateItemRequest.java

```diff
@@ -0,0 +1,25 @@
+package CRM.entity.requests;
+
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@ToString
+public class UpdateItemRequest {
+    //SharedContent fields
+    private Long parentItemId;
+    private Long userId;
+    private String title;
+    private String description;
+
+    //Item fields
+    private Long boardId;
+    private String section;
+    private Long assignedToUserId;
+    private LocalDateTime dueDate;
+    private Integer importance;
+}
```

Comment:
Why not just use ItemRequest?
l don't think we care if there will be a few null fields.
The point in doing the request entity, is to bypass the complicated objects like User, Board, etc.
In this case, all fields are simple: Long, String, etc.